assets/js/invenio_app_rdm/overridableRegistry/mapping.js

```diff
@@ -0,0 +1,3 @@
+export const overriddenComponents = {
+    // "InvenioAppRdm.Deposit.CommunityHeader.layout": () => null,
```

Comment:
@teikmeout if we do not override the CommunityHeader and the line is commented out I recommend removing it before merge

Reply:
Will do!

Reply:
Ok comment removed @ekate. Let me know if you need any more changes.